### PR TITLE
Fix build, glext should not be used with GLEW

### PR DESCRIPTION
**Pull request by gwawiork**
This issue has been noticed by @pierremoreau some time ago
https://github.com/KhronosGroup/OpenCL-CTS/pull/3#issuecomment-486544336

This issue is described here as well: https://stackoverflow.com/questions/65009898/incompatible-function-prototypes-between-glew-h-and-glext-h

Because glew delivers more definitions than glext we should stay with glew.

### CODE REVIEW

File: test_common/gl/setup_x11.cpp [hunk 1]
```diff
@@ -13,15 +13,13 @@
...
-#define GL_GLEXT_PROTOTYPES
 
 #include "setup.h"
 #include "testBase.h"
 #include "harness/errorHelpers.h"
 
 #include <GL/gl.h>
 #include <GL/glut.h>
-#include <GL/glext.h>
 #include <GL/freeglut.h>
```
Comment (bashbaug) [inline test_common/gl/setup_x11.cpp:23]: Could/should we clean up a couple of these other headers while we're at it?

Currently `setup.h` includes `gl_headers.h` which looks like it is intended to include all of the necessary OpenGL headers including those for glut or freeglut.  Do we need to include anything more than glx.h here?

File: test_common/gl/setup_x11.cpp [hunk 2]
```diff
@@ -13,15 +13,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-#define GL_GLEXT_PROTOTYPES
```
Comment (bashbaug) [inline test_common/gl/setup_x11.cpp:16]: This change is good.

It looks setup_win32.cpp defines GL_GLEXT_PROTOTYPES also, even though win32 isn't using glext.h anymore either.  Could we clean it up too?

**Review by bashbaug — DISMISSED** [review]
Mostly LGTM, just a few possible areas for additional cleanup.

Needs a non-Intel reviewer.

File: test_common/gl/setup_x11.cpp (hunk 1 above)
Comment (gwawiork) [reply test_common/gl/setup_x11.cpp]: I am ok. Removed in commit https://github.com/KhronosGroup/OpenCL-CTS/pull/1337/commits/87127a0046aa57381dbe51e729769ef572c4ab76

File: test_common/gl/setup_x11.cpp (hunk 2 above)
Comment (gwawiork) [reply test_common/gl/setup_x11.cpp:16]: removed in https://github.com/KhronosGroup/OpenCL-CTS/pull/1337/commits/4db06f50ee26d58454edceaebb6c076b25b718b0

**Review by bashbaug — COMMENTED** [review]
Thanks, latest version LGTM.  Needs a non-Intel approval.

**Review by kpet — APPROVED** [review]

**Pull request merged**